### PR TITLE
Update target guidance

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: docs/development/manufacturer/config-target-guidance.mdx [hunk 1]
```diff
@@ -6,6 +6,22 @@ sidebar_position: 1
 
 Due to the increase in poorly designed flight controllers, primarily clones of known poorly designed units, Betaflight is now taking a strong stance on adding targets to our cloud build platform.
 
+:::note
+From the 3rd of December 2024, the Betaflight project will no longer accept new flight controller designs using the STM F4 and F7 series microcontroller where there is more than 4 motor outputs.
```
Comment (nerdCopter) [inline docs/development/manufacturer/config-target-guidance.mdx:10]: for 4 motor outputs, F4, F7 still accepted, correct?

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
maybe also note about no longer allowing "SPI" ELRS seems to be the consensus, but needs validation from CEO(s).

**Review by nerdCopter — APPROVED** [review]
- aprroving at `40714749`

**Review by sugaarK — APPROVED** [review]

**Comment by sugaarK** [comment]
> maybe also note about no longer allowing "SPI" ELRS seems to be the consensus, but needs validation from CEO(s).

Perfect

**Pull request merged**